src/pybasic/pybasic.py

```diff
@@ -118,19 +144,14 @@ def fit(self, images: Iterable[ArrayLike]):
 
         ...  # do stuff
 
-    def _initialize_params(self, im_stack: np.ndarray):
-        """Compute initial model parameters.
-
-        Args:
-            im_stack: input images as a stack
-        """
-        ...  # do stuff and set `self.params`
-
-    def predict(self, images: Iterable[np.ndarray]) -> List[np.ndarray]:
+    def predict(
+        self, images: np.ndarray, timelapse: bool = False
+    ) -> Union[Tuple[np.ndarray, np.ndarray], np.ndarray]:
```

Comment:
Change return type to `np.ndarray`?

Reply:
User can grab `flatfield` and `darkfield` separately.

Reply:
Per #14 , we should return
1. `np.ndarray` when `timelapse=False`
2. `(np.ndarray,np.ndarray)` when `timelapse=True`

The first array should be the output images, the second array should be the baseline offsets. This should be made more clear in the documentation. I just put these here as placeholders.

Reply:
Ohh, right!